client/lib/mobile-app/test/index.js

```diff
@@ -0,0 +1,35 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { isMobileApp } from 'lib/mobile-app';
+
+describe( 'mobile-app', () => {
+	test( 'should identify the iOS mobile app', () => {
+		global.navigator = {
+			userAgent:
+				'Mozilla/5.0 (iPhone; CPU iPhone OS 12_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/16B91 wp-iphone/12.1',
+		};
```

Comment:
do we need to reset these global values after teardown?

Reply:
we would need that if this suite contained any other test not setting an explicit user agent. as all the existing tests define a user agent, we don't need to reset the global values.

this doesn't affect other suites since the global scope is reset by the test runner.